impeller/renderer/texture_mipmap.cc

```diff
@@ -0,0 +1,33 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#include "impeller/renderer/texture_mipmap.h"
+#include "impeller/renderer/blit_pass.h"
+#include "impeller/renderer/command_buffer.h"
+
+namespace impeller {
+
+fml::Status AddMipmapGeneration(const std::shared_ptr<Context>& context,
+                                const std::shared_ptr<Texture>& texture) {
+  if (texture->NeedsMipmapGeneration()) {
+    std::shared_ptr<CommandBuffer> mip_cmd_buffer =
+        context->CreateCommandBuffer();
```

Comment:
Rather than creating a new command buffer, we could actually append this blit pass to the existing command buffer before submitting it in `InlinePassContext::EndPass()`

Reply:
Do you think we should do it?  I can add a todo, I probably don't want to fiddle with what works right now.